contrib/golang/common/go/api/capi.go

```diff
@@ -98,6 +98,31 @@ type NetworkCAPI interface {
 	UpstreamInfo(f unsafe.Pointer, infoType int) string
 }
 
+type TcpUpstreamCAPI interface {
+	// UpstreamConnEnableHalfClose upstream conn EnableHalfClose
+	UpstreamConnEnableHalfClose(f unsafe.Pointer, enableHalfClose int)
+	// Header related
+	GetHeader(s unsafe.Pointer, key string) string
```

Comment:
This API is experimental in http plugin, do not need it here.

Reply:
done.